README.md

```diff
@@ -29,6 +29,8 @@ To run the app use `make app-up`
 
 We recommend using [cosmos](https://reactcosmos.org/) for development.
 
+Run CKAN locally (`make up`) and get the Admin user's API Key. Add a test org for development purposes and get the id. Add these values to indicated place in `/metadata-app/src/index.js`.
```

Comment:
Hmm. Shouldn't we be using the user's API key and/or session cookie? Seems wrong to be embedding the admin API key in the app.

Reply:
Yes, in the template we will use the user's `apiKey`. This is just a shim for local development. I updated index.js to make this more clear.